virt/virt-2-5-release-notes.adoc

```diff
@@ -202,8 +206,8 @@ drain if there are virtual machines running on top of them.
 * If the {VirtProductName} storage PV is not suitable for importing a RHV VM, the progress bar remains at 10% and the import does not complete. The VM Import Controller Pod log displays the following error message: `Failed to bind volumes: provisioning failed for PVC`. link:https://bugzilla.redhat.com/show_bug.cgi?id=1857784[(*BZ#1857784*)]
 
 //This defect is ON_QA
-* If you enter the wrong credentials for the RHV Manager while importing a RHV VM, the Manager might lock the admin user account because the `vm-import-operator` tries repeatedly to connect to the RHV API. link:https://bugzilla.redhat.com/show_bug.cgi?id=1854425[*BZ#1854425*]
-
+* If you enter the wrong credentials for the RHV Manager while importing a RHV VM, the Manager might lock the admin user account because the `vm-import-operator` tries repeatedly to connect to the RHV API. link:https://bugzilla.redhat.com/show_bug.cgi?id=1854425[(*BZ#1854425*)]
```

Comment:
There is a difference in the color of the parenthesis in the rendered output based on whether they are around the reference text vs enclosing the entire link (as in some of the other known issues). Can you change the other links as well, just for consistency? Thanks!